src/main/java/me/ironexception/universalfriends/association/Association.java

```diff
@@ -38,8 +41,14 @@ public double getValue() {
     /**
      * Fetch a {@link Association} by its value.
      * Might return null if there is no corresponding
-     * @param value
-     * @return
+     *
+     * <p>
+     * This is like the friendliness level.
+     * </p>
+     *
+     * @param value The friendliness value closest to the actual relation.
```

Comment:
> closest to the actual relation.

Hmm, this function will return `null` for anything that isn't *exactly equal* to an association's assigned value. But how do we word that?

Reply:
I say, we cap off the value in the method and actually prevent that nullification from actually happening. I'll write it up real quick and see what you think.

Reply:
> I say, we cap off the value in the method and actually prevent that nullification from actually happening. I'll write it up real quick and see what you think.

Do that in a separate method. I would, as a developer, only expect this method to return an `Association` if the value matches exactly the association's value. We don't want any magic to happen behind the scenes here.